service/frontend/admin_handler.go

```diff
@@ -240,12 +256,13 @@ func (adh *AdminHandler) Stop() {
 
 func (adh *AdminHandler) DeepHealthCheck(
 	ctx context.Context,
-	request *adminservice.DeepHealthCheckRequest,
+	_ *adminservice.DeepHealthCheckRequest,
```

Comment:
why not remove?

Reply:
this is the interface definition. I want to have the request to have the room to pass in parameters.